projects/api/src/brand/brand.controller.ts

```diff
@@ -0,0 +1,55 @@
+import { Body, Controller, Delete, Get, Param, Patch, Post } from '@nestjs/common';
+import { ApiCreatedResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+
+import BrandService from './brand.service';
+import CreateBrandDto from './dto/create-brand.dto';
+import UpdateBrandDto from './dto/update-brand.dto';
+import Brand from './entities/brand.entity';
+
+@Controller('brands')
+@ApiTags('Brands')
+class BrandController {
+  constructor(private readonly brandService: BrandService) {}
+
+  @Post()
+  @ApiOperation({ summary: 'Create a new brand' })
+  @ApiCreatedResponse({ type: Brand })
+  async create(@Body() createBrandDto: CreateBrandDto): Promise<Brand> {
+    const brand: Brand = await this.brandService.create(createBrandDto);
+    return brand;
+  }
+
+  @Get()
+  @ApiOperation({ summary: 'Get all brands' })
+  @ApiCreatedResponse({ type: [Brand], isArray: true })
```

Comment:
If you put that this ApiCreatedResponse isArray, don't put [Brand] this going to cause problems to Swagger